cassandra-migration-spring-boot-starter/src/test/java/org/cognitor/cassandra/migration/spring/CassandraMigrationAutoConfigurationTest.java

```diff
@@ -31,10 +32,30 @@
 public class CassandraMigrationAutoConfigurationTest {
     private static final String KEYSPACE = "test_keyspace";
 
+    private static Cassandra cassandra;
+    private static final Logger LOGGER = LoggerFactory.getLogger(CassandraMigrationAutoConfigurationTest.class);
+
+    @BeforeClass
+    public static void initDb() {
+        cassandra = new CassandraBuilder()
+                .version("3.11.12")
+                .build();
+        cassandra.start();
+        LOGGER.warn("cassandra started successfully");
```

Comment:
Can we remove these log messages? I assume they were for testing but I don't think that tests need to log messages, especially not at level warning.